Packages/Networking/Networking/Classes/Services/AuthenticationAPIService.swift

```diff
@@ -126,6 +131,49 @@ extension APIService: IAuthenticationAPIService {
 		})
 	}
 	
+	public func login(_ request: Auth_AuthSocialChallengeReq) async -> (Result<Auth_AuthChallengeRes, Error>) {
+		return await withCheckedContinuation({ continuation in
+			let response = clientAuth.login_social_challange(request).response
+			let status = clientAuth.login_social_challange(request).status
+			status.whenComplete({ result in
+				switch result {
+				case .success(let status):
+					if status.isOk {
+						response.whenComplete { result in
+							continuation.resume(returning: result)
+						}
+					} else {
+						continuation.resume(returning: .failure(ServerError(status)))
+					}
+				case .failure(let error):
+					continuation.resume(returning: .failure(ServerError(error)))
+				}
+			})
+		})
+	}
+	
+	
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/18d4719d185fd349dc071bf08a0e7d6ae098fde2/Packages/Networking/Networking/Classes/Services/AuthenticationAPIService.swift#L155"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>AuthenticationAPIService.swift:155</code></td>
    </tr>
  </tbody>
</table>